ledger-service/db-backend/BUILD.bazel

```diff
@@ -33,6 +33,7 @@ da_scala_library(
         "@maven//:org_scalaz_scalaz_core_2_12",
         "@maven//:org_tpolecat_doobie_core_2_12",
         "@maven//:org_tpolecat_doobie_free_2_12",
+        "@maven//:org_tpolecat_doobie_postgres_2_12",
```

Comment:
```suggestion
```

Instead, take the required implicits as arguments, demanding that the service supply them. `selectContracts` uses this to decode signatories and observers.

Reply:
fixed in 32c863255b5c858565e5bf044f8a9938ccdf94c6